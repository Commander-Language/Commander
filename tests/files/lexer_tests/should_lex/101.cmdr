//Test unknown escape characters are interpreted literally (single quote strings)
'x\yz';