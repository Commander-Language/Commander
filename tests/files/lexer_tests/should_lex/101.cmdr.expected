STRINGVAL 2:1
[
STRINGLITERAL 'x\yz' 2:2
]
SEMICOLON ';' 2:7
END_OF_FILE 2:8
